.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Ask a question
+    url: https://github.com/notary-project/nv2/discussions
```

Comment:
typo in the url for notary**-**project:
Should be:
https://github.com/notaryproject/nv2/discussions